lib/proxy/plugin_initializer.rb

```diff
@@ -338,7 +338,7 @@ def load_classes
   end
 
   def validate_settings(plugin, config)
-    result = execute_validators(plugin.plugin_default_settings.keys.map { |k| {:name => :presence, :setting => k} }, config)
+    result = execute_validators(plugin.plugin_default_settings.keys.map { |k| {:name => :presence_allow_empty, :setting => k} }, config)
```

Comment:
I read this, that this is only changing the validation for the default settings.